include/beast/http/parser_v1.hpp

```diff
@@ -62,6 +93,7 @@ class parser_v1
     std::string value_;
     message_type m_;
     typename message_type::body_type::reader r_;
+    std::uint8_t skip_body_ = 0;
```

Comment:
The base class handles the return value of `on_headers`, and it takes care of values 0, 1, and 2:
https://github.com/vinniefalco/Beast/blob/master/include/beast/http/impl/basic_parser_v1.ipp#L902